tox.ini

```diff
@@ -0,0 +1,55 @@
+[tox]
+envlist = py{27,34,35,36}-unit, py36-lint, py36-setup, py{27,34,35,36}-cover, py36-docs
+ 
+[testenv]
+install_command = python -m pip install {opts} {packages}
+deps = 
+  py{27,34,35,36}-unit,py36-lint: mock
+  py{27,34,35,36}-unit,py36-lint: pytest
+  py{27,34,35,36}-unit,py36-lint: pytest-cov
+  py{27,34,35,36}-unit,py36-lint: retrying
+  py{27,34,35,36}-unit,py36-lint: unittest2
+  py{27,34,35,36}-unit,py36-lint,py36-setup,py{27,34,35,36}-cover,py36-docs: -e context/opencensus-context
+  py{27,34,35,36}-unit,py36-lint,py36-docs: -e contrib/opencensus-correlation
+  py{27,34,35,36}-unit,py36-lint,py36-docs: -e .
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-azure
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-dbapi
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-django
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-flask
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-gevent
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-grpc
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-httplib
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-jaeger
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-logging
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-mysql
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-ocagent
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-postgresql
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-prometheus
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-pymongo
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-pymysql
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-pyramid
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-requests
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-sqlalchemy
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-stackdriver
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-threading
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-zipkin
+  py{27,34,35,36}-unit,py36-lint: -e contrib/opencensus-ext-google-cloud-clientlibs
+  py36-lint: flake8
+  py36-setup: docutils
+  py36-setup: pygments
+  py{27,34,35,36}-cover: coverage
+  py{27,34,35,36}-cover: pytest-cov
+  py36-docs: setuptools >= 36.4.0
+  py36-docs: sphinx >= 1.6.3
+
+commands = 
+  py{27,34,35,36}-unit: py.test --quiet --cov={envsitepackagesdir}/opencensus --cov=context --cov=contrib --cov-report= --cov-config=.coveragerc --cov-fail-under=97 tests/unit/ context/ contrib/
```

Comment:
Instead of using the destination folder `{envsitepackagesdir}/opencensus`, can we use the source folder?
The reason for this asking is to avoid run coverage with packages that don't belong to this repo but still under `opencensus` namespace (e.g. opencensus-proto, which is being covered by the coverage report).

Reply:
This is a good point. I will change this to use {envdir} instead to point to the exact source folder in the virtual environment

Reply:
This is a good point. I will use {envdir} instead to point to the virtual environment source directory instead.